core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -133,8 +133,8 @@ public KafkaSchemaRegistry(SchemaRegistryConfig config,
     this.initTimeout = config.getInt(SchemaRegistryConfig.KAFKASTORE_INIT_TIMEOUT_CONFIG);
     this.serializer = serializer;
     this.defaultCompatibilityLevel = config.compatibilityType();
-    this.guidToSchemaKey = new HashMap<Integer, SchemaKey>();
-    this.schemaHashToGuid = new HashMap<MD5, SchemaIdAndSubjects>();
+    this.guidToSchemaKey = new ConcurrentHashMap<>();
```

Comment:
the writes to this happen with a synchronized lock. Is it required to make these concurrent hash maps?

Reply:
The reads are in different threads I believe.